docs/topics/con_key-vertx-concepts.adoc

```diff
@@ -11,7 +11,7 @@ A reactive system, as defined in the link:http://reactivemanifesto.org[reactive
 
 Elastic:: The system remains responsive under varying workload, with individual components scaled and load-balanced as necessary to accommodate the differences in workload. Elastic application deliver the same quality of service regardless of the number of requests they receive at the same time.
 
-Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning beyond a reasonable amount of time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
+Resilient:: The system remains responsive in case any of its individual components fails. Components are isolated from each other, with each individual component being able to quickly recover after failure occurs. Failure of a single component should never prevent other components from functioning properly for a longer time. This prevents cascading failure, where the failure of an isolated components causes other components to become blocked and gradually fail.
```

Comment:
Will fix except the third one; I put the vague specification of time there deliberately because there can indeed be a short time when the services are out. The time, however, mustn't be too long for them to die completely.